singer_sdk/connectors/sql.py

```diff
@@ -969,21 +1039,30 @@ def discover_catalog_entries(self) -> list[dict]:
         result: list[dict] = []
         engine = self._engine
         inspected = sa.inspect(engine)
+        object_kinds = (
+            (reflection.ObjectKind.TABLE, False),
+            (reflection.ObjectKind.ANY_VIEW, True),
```

Comment:
The regression in https://github.com/MeltanoLabs/tap-postgres/pull/528 was probably caused by ignoring views in the updated discovery process.

See https://github.com/MeltanoLabs/tap-postgres/issues/535#issuecomment-2513239541.